AppKit/CPTextView/CPTypesetter.j

```diff
@@ -308,6 +308,12 @@ var CPSystemTypesetterFactory,
 
         }
 
+        // Check if the lineHeight needs to be higher
+        if(theString.charCodeAt(glyphIndex) == 65532 &&
+            64 > currentFontLineHeight)
```

Comment:
do not use this arbitrary constant here.
access the attachment via the textStorage (actually an attributedString).
our first version can assume that the attachment is a CPImage that can be queried for height.

Reply:
even better: do not query the attachment but store the dimensions in private keys of the attributedString upon insertion